scout/parse/variant/frequency.py

```diff
@@ -136,6 +138,7 @@ def parse_mei_frequencies(variant: cyvcf2.Variant) -> Dict:
     update_sv_frequency_from_vcf(mei_frequencies, variant, SWEGEN_HERV_KEYS, "swegen_herv")
     update_sv_frequency_from_vcf(mei_frequencies, variant, SWEGEN_L1_KEYS, "swegen_l1")
     update_sv_frequency_from_vcf(mei_frequencies, variant, SWEGEN_SVA_KEYS, "swegen_sva")
+    update_sv_frequency_from_vcf(mei_frequencies, variant, ["colorsdb_af"], "colorsdb_af")
```

Comment:
Perhaps MEI variants are not annotated on colorsdb and this is an overkill? Is it only SNV and SVs?

Reply:
It's only SNVs and SVs, at least for now.

Reply:
Ok, I'll remove it from the MEIs